### PR TITLE
Add I2S example for M5Core2 in readme

### PR DESCRIPTION
**Pull request by henrivw**
## Motivation and Context
Information

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.M5Stack?pullRequest=237) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.M5Stack?pullRequest=237) before we can accept your contribution.<br/><br/>:x: henrivw [sign now](https://cla.dotnetfoundation.org/nanoframework/nanoFramework.M5Stack?pullRequest=237)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nanoframework/nanoFramework.M5Stack?pullRequest=237) it.</sub>

**Comment by Ellerbach** [comment]
@henrivw please don't forget to sign the license so we can merge ;-) And thanks a lot for your contribution!

File: README.md [hunk 1]
```diff
@@ -366,6 +366,24 @@ var pin5 = M5StickC.GpioController.OpenPin(36, PinMode.Output);
 
 The M5Core, M5Core2, Fire and Atom Lite/Matrix exposes 2 DAC and you can access them thru the `Dac1` and `Dac2` properties. Refer to the [DAC documentation](https://github.com/nanoframework/System.Device.Dac) for more information.
 
+### I2S (sound)
```
Comment (Ellerbach) [inline README.md:369]: In Markdown, you need an extra enter after and before a title

```suggestion
### I2S (sound)

```

File: README.md [hunk 2]
```diff
@@ -366,6 +366,24 @@ var pin5 = M5StickC.GpioController.OpenPin(36, PinMode.Output);
...
+On the M5Core2, wav files can be played using I2S by installing the package [nanoFramework.System.Device.I2s](https://github.com/nanoframework/System.Device.I2S). 
+To play wav files on the M5Core2, use following initialization:
+```csharp
```
Comment (Ellerbach) [inline README.md:372]: In Markdown, the code blocks need to be separated by 1 extra line from the text or anything before and after.
```suggestion

```csharp
```

File: README.md [hunk 3]
```diff
@@ -366,6 +366,24 @@ var pin5 = M5StickC.GpioController.OpenPin(36, PinMode.Output);
...
+
```
Comment (Ellerbach) [inline README.md:373]: And in the code block, the first extra line is not necessary :-)

```suggestion
```

**Comment by Ellerbach** [comment]
@henrivw thanks again and please don't forget to sign the license/cla otherwise we won't be able to merge.

**Pull request merged**